funnel/assets/service-worker-template.js

```diff
@@ -0,0 +1,55 @@
+workbox.core.skipWaiting();
+workbox.core.clientsClaim();
+
+workbox.precaching.precacheAndRoute(self.__precacheManifest);
+
+workbox.routing.registerRoute(new RegExp('/^https?\:\/\/static.*/'), new workbox.strategies.NetworkFirst({
+  "cacheName": "assets"
+}), 'GET');
+
+workbox.routing.registerRoute(new RegExp('/^https?:\/\/hasgeek.com\/*/'), new workbox.strategies.NetworkFirst({
+  "cacheName": "assets"
+}), 'GET');
+
+//For development setup caching of assets
+workbox.routing.registerRoute(new RegExp('/^http:\/\/localhost:3000\/static/'), new workbox.strategies.NetworkFirst({
```

Comment:
On Chrome OS, localhost is `penguin.linux.test`, so need `'/http:\/\/penguin.linux.test:3000\/static/'` covered as well.

Reply:
Service workers are only available on secure origins and localhost.